### ISSUE TITLE
Do we need revisionOverview() in AlertBannerEntityController.php

### ISSUE THREAD

**Issue by andybroomfield** [localgovdrupal/localgov_alert_banner#323]
Theres a deprecation being flagged for renderPlain to renderInIsolation in Drupal 10.3.

We might be able to just remove it though, no other custom entity has this.

**Comment by andybroomfield** [comment]
Add following to the entity and delete controller?
```
 *      "revision-delete" = \Drupal\Core\Entity\Form\RevisionDeleteForm::class,
 *       "revision-revert" = \Drupal\Core\Entity\Form\RevisionRevertForm::class,
 ```

**Comment by andybroomfield** [comment]
Fixed in 1.8.x branch
